lib/internal/callbackify.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = callbackify;
+
+const promises = require('internal/promises');
+const BuiltinPromise = promises.Promise;
+
+function callbackify(fn) {
+  Object.defineProperty(inner, 'name', {
+    writable: false,
+    enumerable: false,
+    configurable: true,
+    value: fn.name + 'Callback'
+  });
+  return inner;
+
+  function inner() {
+    const cb = arguments[arguments.length - 1];
+    if (typeof cb !== 'function') {
+      cb = (err) => { throw err; };
```

Comment:
Ah, only expressions are valid bodies for braceless arrow functions — `err => throw err` doesn't parse.